plugins/org.jkiss.dbeaver.model/OSGI-INF/l10n/bundle_uk.properties

```diff
@@ -0,0 +1,169 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+extension-point.org.jkiss.dbeaver.application.name = \u0414\u043E\u0434\u0430\u0442\u043A\u0438 DBeaver
+extension-point.org.jkiss.dbeaver.pluginService.name = \u0421\u043B\u0443\u0436\u0431\u0430 \u043F\u043B\u0430\u0433\u0456\u043D\u0456\u0432
+extension-point.org.jkiss.dbeaver.dataFormatter.name = \u0424\u043E\u0440\u043C\u0430\u0442\u0435\u0440\u0438 \u0437\u043D\u0430\u0447\u0435\u043D\u044C \u0434\u0430\u043D\u0438\u0445
+extension-point.org.jkiss.dbeaver.aggregateFunction.name = \u0410\u0433\u0440\u0435\u0433\u0430\u0442\u043D\u0456 \u0444\u0443\u043D\u043A\u0446\u0456\u0457
+extension-point.org.jkiss.dbeaver.dataTypeProvider.name = \u041F\u043E\u0441\u0442\u0430\u0447\u0430\u043B\u044C\u043D\u0438\u043A \u0442\u0438\u043F\u0456\u0432 \u0434\u0430\u043D\u0438\u0445
+extension-point.org.jkiss.dbeaver.serialize.name = \u0421\u0435\u0440\u0456\u0430\u043B\u0456\u0437\u0430\u0442\u043E\u0440\u0438 \u043E\u0431\u2019\u0454\u043A\u0442\u0456\u0432
+extension-point.org.jkiss.dbeaver.navigator.name = \u0420\u043E\u0437\u0448\u0438\u0440\u0435\u043D\u043D\u044F \u043D\u0430\u0432\u0456\u0433\u0430\u0442\u043E\u0440\u0430 \u043E\u0431\u2019\u0454\u043A\u0442\u0456\u0432
+
+extension.org.jkiss.dbeaver.DBeaverNature.name = \u041F\u0440\u0438\u0440\u043E\u0434\u0430 DBeaver
```

Comment:
```suggestion
```

Reply:
No need to translate it. Probably, we need to delete all `extension.%` properties at all.